test/integ/mandelbrot/src/main.cpp

```diff
@@ -112,8 +112,7 @@ class MandelbrotKernel
     //-----------------------------------------------------------------------------
     //! Constructor.
     //-----------------------------------------------------------------------------
-    ALPAKA_NO_HOST_ACC_WARNING
-    ALPAKA_FN_ACC MandelbrotKernel()
+    ALPAKA_FN_HOST_ACC MandelbrotKernel()
```

Comment:
This Pull Request enables a test that sets `ALPAKA_ACC_GPU_CUDA_ENABLE` on clang. This uncovered the wrong function attribute because the constructor has to be accessible on the host side as well.